beacon_node/network/src/sync/block_lookups/single_block_lookup.rs

```diff
@@ -328,46 +329,45 @@ impl<E: EthSpec> BlobRequestState<E> {
     }
 }
 
-/// The state of the custody request component of a `SingleBlockLookup`.
+/// The state of the block request component of a `SingleBlockLookup`.
 #[derive(Derivative)]
 #[derivative(Debug)]
-pub struct CustodyRequestState<E: EthSpec> {
+pub struct BlockRequestState<E: EthSpec> {
     #[derivative(Debug = "ignore")]
-    pub block_root: Hash256,
-    pub state: SingleLookupRequestState<DataColumnSidecarList<E>>,
+    pub requested_block_root: Hash256,
+    pub state: SingleLookupRequestState<Arc<SignedBeaconBlock<E>>>,
 }
 
-impl<E: EthSpec> CustodyRequestState<E> {
+impl<E: EthSpec> BlockRequestState<E> {
     pub fn new(block_root: Hash256) -> Self {
         Self {
-            block_root,
+            requested_block_root: block_root,
             state: SingleLookupRequestState::new(),
         }
     }
 }
```

Comment:
Looks like just re-ordering here, maybe revert this so we keep the git blames for those lines for the future.